packages/astro/components/ViewTransitions.astro

```diff
@@ -94,7 +98,7 @@ const { fallback = 'animate', handleForms } = Astro.props;
 				let action = submitter?.getAttribute('formaction') ?? form.action ?? location.pathname;
 				const method = submitter?.getAttribute('formmethod') ?? form.method;
 
-				const options: Options = {};
+				const options: Options = { sourceElement: submitter ?? form };
```

Comment:
sourceElement for Anchors and Forms, see https://chromium.googlesource.com/chromium/src/+/3719b4f51e4919413e8eb08dab383c5278c756a1